Python/Product/EnvironmentsList/PipExtension.xaml.cs

```diff
@@ -76,21 +77,20 @@ internal sealed partial class PipExtension : UserControl {
         }
 
         private void UpgradePackage_CanExecute(object sender, CanExecuteRoutedEventArgs e) {
+            e.Handled = true;

```

Comment:
Nothing wrong with this other than its just cleanup.  However we should have policy to not accept cleanup in the 2.2.1 release